thirdeye/thirdeye-frontend/app/pods/components/alert-report-modal/component.js

```diff
@@ -39,6 +39,17 @@ export default Component.extend({
   showTimePicker: true,
   timePickerIncrement: 30,
 
+  /**
+   * Pre-select all dimensions option if none else available
+   */
+  init() {
+    this._super(...arguments);
+    const dimensionOptions = this.get('dimensionOptions');
+    if (dimensionOptions && dimensionOptions.length === 1 && dimensionOptions[0] === 'All Dimensions') {
```

Comment:
Fyi you can do `dimensionOptions.get('firstObject')` or I think this should work too: `this.get('dimensionOptions.firstObject')`
https://guides.emberjs.com/v2.18.0/object-model/enumerables/#toc_use-of-observable-methods-and-properties

Reply:
Will try that.